typings/index.d.ts

```diff
@@ -2659,6 +2664,7 @@ declare module 'discord.js' {
     intents: BitFieldResolvable<IntentsString, number>;
     ws?: WebSocketOptions;
     http?: HTTPOptions;
+    disableRateLimitQueue?: string[] | ((data: RateLimitData) => boolean | Promise<boolean>);
```

Comment:
It might be helpful to only allow certain values in the array - people wouldn't have to leave their editor to know what could be entered here. It could also help with small mistakes in the value entered (e.g. "guild" vs. "guilds").

Reply:
there are a loooot of possible routes, so I don't think this would be viable and would end up being more limiting than helpful. it would also not allow including ids